example/getting_started/example1/main.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+#include <exception>
+#include <fstream>
+#include <iostream>
+
+#include <powsybl/PowsyblException.hpp>
+#include <powsybl/iidm/Network.hpp>
+
+int main(int argc, char** argv) {
+    try {
+        std::string inputFile = argv[1];
+        std::ifstream inputStream(inputFile.c_str());
+
+        powsybl::iidm::Network network = powsybl::iidm::Network::readXml(inputStream);
+
+        std::string outputFile = argv[2];
+        std::ofstream outputStream(outputFile.c_str());
```

Comment:
Merge these two lines to make the code simpler:
`std::ofstream outputStream(argv[2]);`

Reply:
done